keyboards/nack/keymaps/farfalleflickan/keymap.c

```diff
@@ -46,9 +43,22 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 //  | |----ESC-----|---A--------|---S--------|----D-------|-----F------|-----G------|-----H------|-----J------|-----K------|-----L------|-----Ö------|-----Ä------|--( '* )----|  
        KC_ESC,      KC_A,        KC_S,        KC_D,        KC_F,        KC_G,        KC_H,        KC_J,        KC_K,        KC_L,        KC_SCLN,     KC_QUOT,     KC_NUHS,
 //  | |---SHIFT----|---Z--------|---X--------|----C-------|-----V------|-----B------|-----N------|------M-----|---( ,; )---|---( .: )---|---( -_ )---|----UP------|-BACKSPACE--|  
-       CK_LSFT,     KC_Z,        KC_X,        KC_C,        KC_V,        KC_B,        KC_N,        KC_M,        KC_COMM,     KC_DOT,      KC_SLSH,     CK_UP_PGUP,  CK_BSPC_DEL,
-//  | |---CTRL-----|---ALT------|---META-----|----FN_1----|----------SPACE----------|----FN_2----|----AltGr---|--( <>| )---|---ENETER---|-LEFT-------|---DOWN-----|--RIGHT-----|  
-       KC_LCTL,     KC_LALT,     KC_LGUI,     TT(NUM),     KC_SPC,      KC_NO,       TT(FN),      CK_RALT,     KC_NUBS,     KC_ENT,     CK_LEFT_HOME,CK_DOWN_PGDOWN,CK_RIGHT_END
+       CK_LSFT,     KC_Z,        KC_X,        KC_C,        KC_V,        KC_B,        KC_N,        KC_M,        KC_COMM,     KC_DOT,      KC_SLSH,     KC_UP,       CK_BSPC_DEL,
+//  | |---CTRL-----|---ALT------|---META-----|----FN_1----|----------SPACE----------|----FN_2----|----AltGr---|--( <>| )---|---ENTER----|-LEFT-------|---DOWN-----|--RIGHT-----|  
+       KC_LCTL,     KC_LALT,     KC_LGUI,     MO(NUM),     KC_SPC,      KC_NO,       MO(FN),      CK_RALT,     KC_NUBS,     KC_ENT,      KC_LEFT,     KC_DOWN,     KC_RIGHT
+//   \|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|         
+    ),
+    [NOGUI] = LAYOUT(
+/*   __________________________________________________________________________________________________________________________________________________________________________
+    | \            \            \            \            \            \            \            \            \            \            \            \            \            \        */                                                                                                                                                                 
```

Comment:
```suggestion
    | \            \            \            \            \            \            \            \            \            \            \            \            \            \        */
```
Please remove all trailing whitespace.